src/components/Pdf/PdfViewer.tsx

```diff
@@ -80,10 +103,19 @@ export function PdfViewer({ onTooltipClick, file, onDownloadPdf, onDownloadJson,
               Download As
```

Comment:
Icon instead of button with text just as in the charts. Maybe copy the dropdown component from there